newsletter-may-2021.rst

```diff
@@ -0,0 +1,80 @@
+.. post:: May 5, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from May.
+
+Read the Docs newsletter - May 2021
+===================================
+
+Welcome to a new edition of our monthly newsletter, in which we would like to
+openly share with you the most relevant updates of Read the Docs,
+offer a summary of what new features we shipped to our users
+during the previous month,
+and share what things we will be focusing on in the near future.
```

Comment:
```suggestion
and share what we'll be focusing on in the near future.
```

Reply:
Sorry for all the edits -- just trying to make things more concise :)

Reply:
They're very welcome :)